### PR TITLE
[BUGFIX] Fixes CI + other tests

### LINKED ISSUE #75: [BUG] Fix tests
Fix various tests:

- [ ] `tests/scripts/test_coarse_to_fine.py::test_coarse_to_fine`
- [ ] `tests/mappings/test_sparse_barycenter.py::test_fugw_barycenter`

### PR DESCRIPTION
**Pull request by pbarbarant**
Closes #75 and improves testing of both `FUGWBarycenter` and `FUGWSparseBarycenter`.

### CODE REVIEW

**Comment by bthirion** [comment]
I'll review when the CI is green.

**Comment by pbarbarant** [comment]
> I'll review when the CI is green.

Fixed.

File: tests/mappings/test_sparse_barycenter.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "device, callback",
     product(devices, callbacks),
 )
-def test_fugw_barycenter(device, callback):
+def test_fugw_sparse_barycenter(device, callback):
     np.random.seed(0)
```
Comment (bthirion) [inline tests/mappings/test_sparse_barycenter.py:23]: Could you add a one line docstring to make clear what is tested here ?

**Review by bthirion — COMMENTED** [review]
LGTM overall.

**Comment by pbarbarant** [comment]
I also added extra assertions on the outputs of FUGW barycenters.

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Pull request merged**